src/v2/Apps/Collect/Routes/Collection/Components/Header/index.tsx

```diff
@@ -65,7 +66,9 @@ export const CollectionHeader: React.FC<CollectionHeaderProps> = ({
 
         {collection.description && (
           <Column span={6}>
-            <HTML html={collection.description} variant="sm" />
+            <HTML variant="sm">
+              <ReadMore maxChars={750} content={collection.description} />
+            </HTML>
```

Comment:
(Really need to get some code snippet output into our palette storybooks; had to open repo and dig in)